### PR TITLE
Update to timed v0.13.1

### PR DESCRIPTION
**Pull request by hairmare**
* [x] bump version in chart
* [x] configure a sensible default for `DJANGO_TRACKING_REPORT_VERIFIED_CHANGES`

### CODE REVIEW

**Comment by sbor23** [comment]
sensible default is included in the timed-backend settings

**Review by strixBE — COMMENTED** [review]
why `0.13` and not `0.13.0`?

**Comment by strixBE** [comment]
> sensible default is included in the timed-backend settings

Does this mean our test fails with a false-negative and we ignore it or do we need to take action?

**Comment by sbor23** [comment]
> > sensible default is included in the timed-backend settings
> 
> Does this mean our test fails with a false-negative and we ignore it or do we need to take action?

I means that we changed that yesterday and the CI probably has to be run again to pick up the changes.

**Comment by sbor23** [comment]
> why `0.13` and not `0.13.0`?

This is the actual tag. But we're tagging the 0.13.1 bugfix release in a matter of hours so this should be fixed soon. ;)

File: charts/timed/Chart.yaml [hunk 1]
```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: timed
 type: application
 description: Chart for Timed application
-version: 0.2.2
-appVersion: v0.12.1
+version: 0.3.0
+appVersion: v0.13
```
Comment (strixBE) [inline charts/timed/Chart.yaml]: please bump to v0.13.1 directly

**Review by strixBE — CHANGES_REQUESTED** [review]

**Comment by hairmare** [comment]
> why 0.13 and not 0.13.0?

It should have been 0.13.0 and we're fixing this going forward.

I just updated the PR to use 0.13.1.

**Comment by strixBE** [comment]
waiting for merge of https://github.com/adfinis-sygroup/timed-frontend/pull/423 and pushing new image

**Review by strixBE — APPROVED** [review]
lgtm

**Pull request merged**